src/log.js

```diff
@@ -1,8 +1,5 @@
 let log = require('taskcluster-lib-log');
 
-let env = process.env.NODE_ENV;
-if (!env) {
-  throw new Error('You must have an environment');
-}
+let env = process.env.NODE_ENV || "test";
```

Comment:
test is not the right thing to have as default.
```javascript

let env;
if (!process.env.NODE_ENV) {
  env = ""
} else {
  env = '-' + process.env.NODE_ENV.trim();
}
module.exports = log('aws-provisioner' + env); 
```

I'd rather that we just always set NODE_ENV...